boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -368,23 +362,19 @@ zephyr_udc0: &usbhs {
 
 			boot_partition: partition@0 {
 				label = "mcuboot";
-				reg = <0x00000000 DT_SIZE_K(64)>;
+				reg = <0x00000000 DT_SIZE_K(256)>;
```

Comment:
Why does MCU boot partition need to increase in size?

Reply:
I'm increasing the MCUBoot partition size here to mirror some of our other platforms, where MCUBoot partitions are larger. with certain options, MCUBoot can exceed 64KB of flash space. I've tried a few settings and I've yet to see it exceed 128KB though, so I will update this PR with that size.